samples/sensor/stm32_temp_sensor/boards/nucleo_f401re.overlay

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2021 Eug Krashtan
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Application overlay for creating temperature sensor device instance
+ */
+
+/ {
+	stm-temp {
+		label = "DIE_TEMP";
+		compatible = "st,stm32-temp";
+		io-channels = <&adc1 16>;
+		status = "okay";
+	};
+};
+
+&adc1 {
+	status = "okay";
+};
```

Comment:
adc1 is already part of nucleo_f401re default configuration (`nucleo_f401re.dts`)
.overlay is added on top of this default configuration, so this is redundant.